tck/src/main/java/org/reactivestreams/tck/PublisherVerification.java

```diff
@@ -128,7 +128,6 @@ public long maxElementsFromPublisher() {
 
   /**
    * Override and return {@code true} in order to skip executing tests marked as {@code Stochastic}.
-   * Such tests MAY sometimes fail even though the impl
```

Comment:
Good point. I'll add:

"Stochastic in this case means that the Rule is impossible or infeasible to deterministically verify— usually this means that this test case can yield false positives ("be green") even if for some case, the given implementation may violate the tested behaviour."